editor/src/app/login/login.component.ts

```diff
@@ -27,9 +27,11 @@ import { PopupService } from '../popup/popup.service';
     <ng-template #logInForm>
       <div *ngIf="appState.isBertaHosting" class="form-group social-login">
         <a href="{{ appState.loginUrl }}?remote_redirect={{ appState.authenticateUrl }}&amp;provider=facebook" class="button facebook">
-          Log in with Facebook</a>
+        <bt-icon-facebook></bt-icon-facebook>
+        <p>Log in with Facebook</p></a>
         <a href="{{ appState.loginUrl }}?remote_redirect={{ appState.authenticateUrl }}&amp;provider=google" class="button google">
-          Log in with Google</a>
+        <img src="../../../../engine/layout/Google.png" alt="G">
```

Comment:
Check how other images are referenced in the login component. See line [21](https://github.com/berta-cms/berta/pull/300/files#diff-7f57b5386344a10b7e842dc23fe57536R21).